### PR TITLE
French support (fr_fr.json) & various strings updates (en_us.json)

### PR DESCRIPTION
**Pull request by DracoNocte**
Added French translations for all current blocks (atomizer, liquefier...), items (fertilizer, obsidian breaker...) and UI elements (JEI, infos...).

### CODE REVIEW

File: src/main/resources/assets/alchemistry/lang/fr_fr.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  "alchemistry.jei.atomizer": "Atomiseur",
+  "alchemistry.jei.liquifier": "Liquéfacteur",
+  "alchemistry.jei.fission": "Chambre à fission (multiblock)",
+  "alchemistry.jei.fusion": "Chambre à fusion (multiblock)",
+
+  "itemGroup.alchemistry": "Alchemistry",
+
+  "item.alchemistry.fertilizer.tooltip": "Peut être utilisé comme de la poudre d'os.",
+  "item.alchemistry.obsidian_breaker.tooltip": "Cliquez droit sur un bloc d'obsidienne pour la récolter immédiatement. Cet objet est à usage unique.",
+  "item.alchemistry.milk.tooltip": "All I do is milk.",
```
Comment (Mazdallier) [inline src/main/resources/assets/alchemistry/lang/fr_fr.json:37]: Tout ce que je fais est du lait

File: src/main/resources/assets/alchemistry/lang/fr_fr.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  "block.alchemistry.fission_controller": "Contrôleur de chambre à fission",
+  "block.alchemistry.fission_core": "Cœur de chambre à fission",
+  "block.alchemistry.fission_casing": "Coffrage de chambre à fission",
+  "block.alchemistry.fusion_controller": "Contrôleur de chambre à fusion",
+  "block.alchemistry.fusion_core": "Cœur de chambre à fusion",
+  "block.alchemistry.fusion_casing": "Coffrage de chambre à fusion",
+
+  "item.alchemistry.mineral_salt": "Sel minéral",
+  "item.alchemistry.fertilizer": "Engrais",
+  "item.alchemistry.obsidian_breaker": "Brise-obsidienne",
```
Comment (Mazdallier) [inline src/main/resources/assets/alchemistry/lang/fr_fr.json:19]: Casse-obsidienne

File: src/main/resources/assets/alchemistry/lang/fr_fr.json (hunk 1 above)
Comment (DracoNocte) [reply src/main/resources/assets/alchemistry/lang/fr_fr.json:37]: This tooltip is meant to be an easter egg. Therefore there is no reason to translate it.

File: src/main/resources/assets/alchemistry/lang/fr_fr.json (hunk 2 above)
Comment (DracoNocte) [reply src/main/resources/assets/alchemistry/lang/fr_fr.json:19]: > Casse-obsidienne
@Mazdallier

"Brise-obsidienne" is already an accurate translation of "Obsidian Breaker".
Why do you want it to be replaced with a synonym ? Is it different according to you ?

**Comment by ghost** [comment]
It took me far too long, but this has been added to v1.16.4-1, thank you

**Pull request closed without merging**